### PR TITLE
Introduce the openSLP test

### PR DESCRIPTION
**Pull request by pdostal**
Hello, this time I have openSLP test:

- Related ticket: [poo#51581](https://progress.opensuse.org/issues/51581)
- Needles: No need
- Verification runs:
   * [SLES12SP1](http://pdostal-server.suse.cz/tests/4096) [SLES12SP2](http://pdostal-server.suse.cz/tests/4099)
   * [SLES12SP3](http://pdostal-server.suse.cz/tests/4098) [SLES12SP4](http://pdostal-server.suse.cz/tests/4097)
   * [SLES15](http://pdostal-server.suse.cz/tests/4094) [SLES15SP1](http://pdostal-server.suse.cz/tests/4095)
   * [Tumbleweed](http://pdostal-server.suse.cz/tests/4092) [LEAP15.2](http://pdostal-server.suse.cz/tests/4093)

### CODE REVIEW

File: tests/console/slp.pm [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# SUSE's Apache regression test
```
Comment (foursixnine) [inline tests/console/slp.pm]: Apache? :)

**Comment by okurz** [comment]
s/Introduce/Introduce/

**Comment by pdostal** [comment]
Typos fixed, verification runs restarted, everything seems to be OK from my side.

**Comment by dzedro** [comment]
passed also on 12sp5 https://openqa.suse.de/tests/3291410#step/slp/36

**Comment by Vogtinator** [comment]
Fails on JeOS: https://openqa.opensuse.org/tests/1019620#step/slp/19

**Pull request merged**